From 16fc60d25faf0637b603bea2daa45d244c44a670 Mon Sep 17 00:00:00 2001
From: Daniel Rosenberg <drosen@google.com>
Date: Wed, 30 Sep 2015 20:44:16 -0700
Subject: [PATCH 630/634] staging: ion: Add X86 dependency for
 ION_POOL_CACHE_POLICY

ION_POOL_CACHE_POLICY uses x86 specific commands.
Only allow it to be used for x86.

Signed-off-by: Daniel Rosenberg <drosen@google.com>
Change-Id: Ic66e175fa41486b0b5f77872f2193add77cbe66f
---
 drivers/staging/android/ion/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/android/ion/Kconfig b/drivers/staging/android/ion/Kconfig
index 9ce6c36..c198e6f 100644
--- a/drivers/staging/android/ion/Kconfig
+++ b/drivers/staging/android/ion/Kconfig
@@ -35,7 +35,7 @@ config ION_TEGRA
 
 config ION_POOL_CACHE_POLICY
 	bool "Ion set page pool cache policy"
-	depends on ION
+	depends on ION && X86
 	default y if X86
 	help
 	  Choose this option if need to explicity set cache policy of the
-- 
1.9.1

